### PR TITLE
Added Support for LogLevelPrintable

### PR DESCRIPTION
**Pull request by thingdeux**
Added support for LogLevelPrintable and tests.

### CODE REVIEW

File: BoneKit/Classes/Utils/DebugLog.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+public enum DebugLevel: UInt {
+    case error = 0
+    case info
+    case debug
+    case verbose
+    /// An overwhelming level of logging for when you absolutely have to log
+    /// everything including the kitchen sink.
+    case excessive
+}
+
```
Comment (mnorth719) [inline BoneKit/Classes/Utils/DebugLog.swift:21]: Formatting - No space after DebugLevel

File: Example/Tests/DebugLogTests.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            XCTAssertEqual(shouldPrintInfo, didPrintInfo)
+            
+            let didPrintDebug = printLog("", atOrAbove: .debug)
+            XCTAssertEqual(shouldPrintDebug, didPrintDebug)
+            
+            let didPrintVerbose = printLog("", atOrAbove: .verbose)
+            XCTAssertEqual(shouldPrintVerbose, didPrintVerbose)
+            
+            let didPrintExcessive = printLog("", atOrAbove: .excessive)
+            XCTAssertEqual(shouldPrintExcessive, didPrintExcessive)
```
Comment (mnorth719) [inline Example/Tests/DebugLogTests.swift:70]: Formatting - Trim extra empty lines

**Review by mnorth719 — COMMENTED** [review]
Looks good, just a couple of minor formatting comments

**Comment by thingdeux** [comment]
@mnorth719  - Updated per code review.

**Review by mnorth719 — APPROVED** [review]

**Pull request merged**